s3tests_boto3/functional/test_s3select.py

```diff
@@ -612,10 +612,6 @@ def test_when_than_else_expressions():
     bucket_name = "test"
     upload_csv_object(bucket_name,csv_obj_name,csv_obj)
 
-    res_s3select = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name,'select  case when ((4*3)==(12)) than "case_1_2" else "case_2_1" end from stdin where (3*3==9);')  ).replace("\n","")          
```

Comment:
why is this test removed?

Reply:
the test is wrong , the assert for that test is wrong , it should compare against result of other query.
(in later PR https://github.com/ceph/s3-tests/pull/375 , it is fixed)